### DISCUSSION TITLE
rAudio 20230522: Error Messages at Update

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1306] [Fixed Issues]
At the end of the update I got
```
grep: /srv/http/data/system/localbrowser.conf: No such file or directory 
convert: invalid argument for option '-rotate': -gravity @ error/convert.c/ConvertImageCommand/2683.
```
The first message is no surprise because I don't use a local browser.
The second one seems to originate from imagemagick, cause unknown.

**Comment by ceejayemm** [comment]
Not seeing this after updating 2 RPi (3A and 3B+) to 20230522

Chris

**Comment by klassikmann** [comment]
rAudio 20230523: Still present

**Comment by klassikmann** [comment]
rAudio 20230525: Fixed
